ivy/functional/ivy/creation.py

```diff
@@ -685,14 +854,20 @@ def full(
 @to_native_arrays_and_back
 @handle_out_argument
 @handle_nestable
-def from_dlpack(x: Union[ivy.Array, ivy.NativeArray]) -> ivy.Array:
+def from_dlpack(
+    x: Union[ivy.Array, ivy.NativeArray],
+    out: Optional[ivy.Array] = None,
```

Comment:
`out` argument should be keyword-only, with a `*` added above it. This is the case for all functions (I may have missed this for others).